sw/device/lib/testing/pinmux_testutils.c

```diff
@@ -129,3 +129,45 @@ uint32_t pinmux_testutils_get_testable_gpios_mask(void) {
     return 0xffffffff;
   }
 }
+
+uint32_t pinmux_testutils_read_strap_pin(dif_pinmux_t *pinmux, dif_gpio_t *gpio,
+                                         dif_gpio_pin_t io,
+                                         top_earlgrey_muxed_pads_t pad) {
+  // Turn off the pull enable on the pad and read the IO.
+  dif_pinmux_pad_attr_t attr = {.flags = 0};
```

Comment:
nit: did you set `flags` because you don't like the form below?
```
dif_pinmux_pad_attr_t attr = {0};
```

Reply:
IMOH, being verbose about `.flags` communicates my intent better than just `{0}`.